### PR TITLE
added my name to README.md

### PR DESCRIPTION
**Pull request by arun11meena**
ping @leosartaj

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Lines starting with `*` are automatically rendered as bullet points and phrases
 * Rishabh Daal [rd13123013](http://github.com/rd13123013)
 * Sartaj Singh [leosartaj](http://github.com/leosartaj)
 * Shivam Tyagi [shivamT95](http://github.com/shivamT95)
-
+* Arun Meena [arun11meena](http:/github.com/arun11meena)
```
Comment (leosartaj) [inline README.md]: Please add a new line here.

**Comment by arun11meena** [comment]
Have a look @leosartaj

**Comment by arun11meena** [comment]
Have a look @leosartaj

File: README.md [hunk 2]
```diff
@@ -32,4 +33,4 @@ Lines starting with `*` are automatically rendered as bullet points and phrases
 * You should also go through the [curated list of git resource](https://help.github.com/articles/good-resources-for-learning-git-and-github/)
   provided by github to find other resouces to fit your need.
 * Here's a fun game based on git, [git-game](https://github.com/git-game/git-game).
-* A list of cool features of Git and Github http://git.io/sheet.
+ A list of cool features of Git and Github http://git.io/sheet.
```
Comment (leosartaj) [inline README.md]: Bullet point.

**Comment by arun11meena** [comment]
ping @leosartaj

**Comment by leosartaj** [comment]
Looks good. Merging.

**Pull request merged**